src/pages/Main.js

```diff
@@ -1,66 +1,189 @@
-import { useState } from 'react'
+import { useEffect, useState } from 'react'
 import Display from "../components/Display"
 import Filters from '../components/Filters'
 let rawData = require("../data/data.json")
 
 const Main = () => {
+    const [data, setData] = useState(rawData);
     const [tag, setTag] = useState("Please select a category");
```

Comment:
Do not use 'Please select a category' as tag's value, render it in the select component